ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -206,8 +203,7 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
       _ => {
         fromNullable(this.navigationKeyFrom).foldL(
```

Comment:
Since we are, what about this refactoring to remove some boiler plate?
```typescript
private handleBackPress = () => {
    this.props.navigateBack(this.newMethodAdded ? undefined : this.navigationKeyFrom);
    return true;
  };
```

Reply:
Changed in ccd6cf20978697481fadf8d6aaefcada6183d59d but following the previous code should pop the screens only if  `this.newMethodAdded && this.navigationKeyFrom !== undefined`:

```typescript
    const shouldPop =
      this.newMethodAdded && this.navigationKeyFrom !== undefined;

    this.props.navigateBack(shouldPop ? this.navigationKeyFrom : undefined);
    return true;
```
Seems right to you?